docs/contributing/database-migrations/mssql.md

```diff
@@ -0,0 +1,82 @@
+---
+sidebar_position: 1
+---
+
+# MSSQL
+
+:::info
+
+For instructions on how to apply database migrations, please refer to the
+[Getting Started](../../getting-started/server/database/mssql/index.md) documentation.
+
+:::
+
+:::tip
+
+We recommend reading [T-SQL Code Style](../code-style/sql.md) since it has a major impact in how we
+write migrations.
+
+:::
+
+## SQL database project
+
+The separate database definitions in `src/Sql/.../dbo` serve as a "master" reference for the
+intended and final state of the database at that time. This is crucial because the state of database
+definitions at the current moment may differ from when a migration was added in the past. These
+definitions act as a lint and validation step to ensure that migrations work as expected, and the
+separation helps maintain clarity and accuracy in database schema management and synchronization
+processes.
+
+Additionally, a
+[SQL database project](https://learn.microsoft.com/en-us/azure-data-studio/extensions/sql-database-project-extension-sdk-style-projects)
+is in place; however, instead of using the auto-generated migrations from
+[DAC](https://learn.microsoft.com/en-us/sql/relational-databases/data-tier-applications/data-tier-applications?view=sql-server-ver16),
+we manually write migrations. This approach is chosen to enhance performance and prevent accidental
+data loss, which is why we have both a `sqlproj` and standalone migrations.
+
+## Modifying the database
+
+In accordance with the tenets of [Evolutionary Database Design](./edd.mdx) every change must be
+considered as split into two parts:
+
+1. A backwards-compatible transition migration
+2. A non-backwards-compatible final migration
+
+It is likely that a change does not require a non-backwards-compatible end phase e.g. all changes
+may be backwards-compatible in their final form; in that case, only one phase of changes is
+required. With the use of beta testing, partial roll-outs, [feature flags](../feature-flags.md),
+etc. the often-chosen path is to spread a change across several major releases with a calculated
+future state that can perform a "cleanup" migration that is backwards-compatible but still
+represents an overall-_incompatible_ change beyond the boundaries of what we need for individual
```

Comment:
⛏️ Our communication guidelines indicate we should avoid italics and prefer bold because international fonts don't render italics well.

```suggestion
represents an overall-**incompatible** change beyond the boundaries of what we need for individual
```

Reply:
It's in the [Technical Writers Style Guide](https://bitwarden.atlassian.net/wiki/x/D4BTN).